storage/flux/reader.go

```diff
@@ -12,9 +12,11 @@ import (
 	"github.com/influxdata/flux/memory"
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/values"
-	"github.com/influxdata/influxdb/flux/stdlib/influxdata/influxdb"
+	query "github.com/influxdata/influxdb/flux/stdlib/influxdata/influxdb"
```

Comment:
I tried moving the relevant file (`storage.go`) out of `influxdb` and into `query`, but hit circular reference errors. Decided to leave it alone for now